Jenkinsfile

```diff
@@ -70,11 +70,5 @@ def golint() {
 }
 
 def build() {
-  docker.withRegistry('https://registry.internal.exoscale.ch') {
-    def image = docker.image('registry.internal.exoscale.ch/exoscale/golang:1.10')
-    image.inside("-u root --net=host -v ${env.WORKSPACE}/src:/go/src/github.com/exoscale/zlocker") {
-      sh 'cd /go/src/github.com/exoscale/zlocker && dep ensure'
-      sh 'cd /go/src/github.com/exoscale/zlocker && CGO_ENABLED=0 go build -ldflags "-s"'
-    }
-  }
+  docker.build('registry.internal.exoscale.ch/exoscale/zlocker:latest').withRun('--net=host')
```

Comment:
https://github.com/exoscale/exoip/blob/master/Jenkinsfile#L84-L109

Reply:
iirc you have to have the two steps in order to produce the local binary for the .deb and the container image.